docs/MAINTAINING.md

```diff
@@ -0,0 +1,40 @@
+# Maintaining LoopBack
+
+Congratulations! Since you're reading this page, you are probably already a
+maintainer of this repo or close to be one. Thank you for your contributions so
+far!
+
+## Why do I want to be a maintainer?
+
+- Greater influence on LoopBack's future direction
+- Commit (write) rights to `loopback-next` repo
+- Ability to review and land pull requests, edit/categorize/close issues
+- Access to a private Gitter room for LoopBack maintainers
```

Comment:
Gitter or Slack?

Reply:
Actually, I meant to remove it.  Thanks for catching that.